exercises/Ex5Christmas.js

```diff
@@ -0,0 +1,22 @@
+function nextChristmas(){ 
```

Comment:
nice, but too much done.

I would solve as follows: 
1. find this year's Christmas date (xmsDate).
2. If TODAY < xmsDate then return xmsDate - TODAY.
3. find next year's Christmas date, return xmsDate - TODAY.

Reply:
Refactored:

find this year's Christmas date (xmsDate).
If TODAY >= xmsDate then find next year's Christmas date (xmsDate).
return xmsDate - TODAY.